content/examples/databricks-cluster.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Using sparklyr with Databricks"
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+    toc_float:
+      collapsed: false
+params:
+  width: 600
+---
+
+## Overview
+
+This documentation demonstrates how to use `sparklyr` with RStudio Server Pro
+and a Databricks cluster with Apache Spark.
```

Comment:
Do we want something in here about RSC? Is that its own, separate resource?

Reply:
Separate resource that we can add later, IMO. For now I added a short note at the top of the best practices on the front page:

> Install RStudio Connect and RStudio Package Manager outside of the Databricks cluster so that they are not limited to the resources or ephemeral nature of Databricks clusters

And I'll see about clarifying or condensing those bullets per your other review comment. 👍